### DISCUSSION TITLE
Mobile apps can't pop up to remind you of new messages

### DISCUSSION THREAD

**Discussion by zs2766** [RocketChat/Rocket.Chat.ReactNative#4066] [Q&A] [answered]
I have installed and tested both Android and Apple apps, and the window notification cannot pop up when new information comes. I have opened the notification option of all information in the APP, but it does not work at all. Please solve this problem, thank you!

**Comment by diegolmello** [comment, marked as answer]
Hey, I think you're referring to push notifications, right?
There can be several reasons for a push notifications not to be sent to our app.
The push quota might have been achieved, the message doesn't follow the [notification workflow](https://whimsical.com/notification-workflow-PRwN4MWNsxSwqHjHXaPUuC), etc
